config/schemas/road.json

```diff
@@ -0,0 +1,37 @@
+{
+	"type":"object",
+	"$schema": "http://json-schema.org/draft-04/schema",
+	"title" : "VCMI road format",
+	"description" : "Format used to define new roads in VCMI",
+	"required" : [ "text", "shortIdentifier", "tilesFilename", "moveCost" ],
+
+	"additionalProperties" : false,
+	"properties":{
+		"index" : 
+		{
+			"type": "number",
+			"description": "Internal, do not use"
+		},
+		"text":
+		{
+			"type": "string",
+			"description": "Human-readable name of the road"
+		},
+		"shortIdentifier":
+		{
+			"type": "string",
+			"description": "Two-letters unique indentifier for this road. Used in map format"
+		},
+		"tilesFilename":
```

Comment:
Just a suggestion to name it tilesGraphics. tilesFilename also looks good I am only afraid that we do not work with files directly but with some kind of resources abstraction. So do we use term file in configs anywhere?

Reply:
From consistency point of view yes, "tilesGraphics" should be more suitable.
Not sure about readability though. "File" clearly states that modder needs file with terrain tile. "Graphics" is more abstract.